### ISSUE TITLE
What has already taken place at `TracePostExecution` ?

### ISSUE THREAD

**Issue by OlivierBBB** [Consensys/besu-sequencer-plugins#870]
E.g.
> **Question 1.** If you have an aborted CALL: has the recipient's warmth been switched on at `TracePostExecution` ?

> **Question 2.** If you have an unaborted CALL: has the recipient's warmth been switched on at `TracePostExecution` and has the balance transfer been done at that point ?

> **Question 3.** Has the $\texttt{gasCost} + \max \{ \mu[0], \texttt{63/64ths} \}$ been deducted from the caller's gas ?

**Comment by lu-pinto** [comment]
>Question 1. If you have an aborted CALL: has the recipient's warmth been switched on at TracePostExecution ?

Yes, it is done between `tracePreExecution` and `tracePostExecution` in [AbstractCallOperation.java#L182](https://github.com/hyperledger/besu/blob/2158a6829676c75931ef5522f06332965fa6c4b6/evm/src/main/java/org/hyperledger/besu/evm/operation/AbstractCallOperation.java#L182) while executing the CALL operation before any checks. There's just one check before it which I can't decode, that returns an `UNDERFLOW_RESPONSE`. It looks like defensive programming to me when there's not enough stack items, see if it's relevant to you. Does it go into your definition of aborted?

>Question 2. If you have an unaborted CALL: has the recipient's warmth been switched on at TracePostExecution and has the balance transfer been done at that point ?

As per question 1 answer yes, address has been warmed. Regarding balance transfer it is done only after `tracePostExecution` of the parent has been done, and while the child frame is executing. It's done in the `start()` method called by the child frame: [MessageCallProcessor.java#L72](https://github.com/hyperledger/besu/blob/2158a6829676c75931ef5522f06332965fa6c4b6/evm/src/main/java/org/hyperledger/besu/evm/processor/MessageCallProcessor.java#L72)

>Question 3. Has the gasCost + max μ [ 0 ] , 63/64ths been deducted from the caller's gas ?

You are talking about the changes after the [Tangerine Whistle fork](https://www.evm.codes/#f1?fork=cancun) right?
From what I can understand from the code, this gas cost is deducted during [AbstractCallOperation.execute(...)](https://github.com/hyperledger/besu/blob/2158a6829676c75931ef5522f06332965fa6c4b6/evm/src/main/java/org/hyperledger/besu/evm/operation/AbstractCallOperation.java#L183-L206) which would be before `tracePostExecution` of the parent frame. What I understand here is that parent gets deducted [gas (1 -1/64) and more](https://github.com/hyperledger/besu/blob/2158a6829676c75931ef5522f06332965fa6c4b6/evm/src/main/java/org/hyperledger/besu/evm/operation/AbstractCallOperation.java#L187) which I think, if I'm not missing anything, looks like a bug actually. In the case where we have a failure condition (contract call depth > 1024 or no available balance on sender) we actually increment the right gas back on line 202. And I checked, `gasAvailableForChildCall` uses the right gasCap (`gas (1-1/64)`) mentioned in the specification.
